src/_data/featured_work.yml

```diff
@@ -32,3 +32,17 @@
     - Project Management
     - Human-centered Design
     - Software Development
+
+- title: Upgrade Scenario Planning Model (SPM)
+  description: Collaborated with a diverse team to update and enhance the SPM transportation model by developing database queries and a user interface for the Southern California Association of Governments (SCAG).
```

Comment:
SCAG is mentioned in the Data+Donuts section without spelling it out, but the D+D section comes above this one. Sooo... ideally the D+D section spells out `Southern California Association of Governments (SCAG)` as it's the first reference on the page, and the rest following can just be SCAG.

Reply:
Yep I think this change makes sense